src/components/Expenses/ExpensesChart.tsx

```diff
@@ -1,7 +1,8 @@
 import React from "react";
+import { expenseData } from "../../App";
 import Chart from "../Chart/Chart";
 
-const ExpensesChart = (props) => {
+const ExpensesChart = (props: { expensesData: expenseData[] }) => {
```

Comment:
If you create a type for the props and use that, you'll be able to directly destructure the props and get their variables, through the provided type.